app/queries/get_web_content_items.rb

```diff
@@ -1,79 +0,0 @@
-module Queries
```

Comment:
yay for removing this, only need to refactor one validator and we can lose the arel helpers.